### PR TITLE
[NEW ICON]: Tally Prime and Canva Icons

### LINKED ISSUE #74: [NEW ICON]: TALLY Prime
### Skill Name

Tally Prime - Accountings

### Why?

its an indian based accounting software , so i need to add that in my profile and its too famous as same as sap https://tallysolutions.com/tally-prime/

### Reference Image

Tally-Prime

### Code of Conduct Acknowledgment

- [X] I have read and agree to the Code of Conduct.

### LINKED ISSUE #184: [NEW ICON] Canva
### Skill Name

Canva

### Why?

It's an popular design tool.

### Reference Image

canva-seeklogo com

### LINKED ISSUE #358: [NEW ICON] Skill Name
### Skill Name

Canva

### Why?

Image making

### Reference Image

_No response_

### LINKED ISSUE #805: [NEW ICON] Canva
### Skill Name

Canva

### Why?

It is most popularly used design application used by designers and also for presentations after Adobe and MS PowerPoint.

### Reference Image

image

### PR DESCRIPTION
**Pull request by josemateuss**
# Add New Icon(s)

## Icon Details

| Icon ID | Icon |
|---------|------|
| tallyprime | tallyprime |
| canva-dark | canva-dark |
| canva-light | canva-light |

## Notes

- [Tally Prime](https://tallysolutions.com/tally-prime/) is an Indian based accounting software, it's too famous as same as SAP, as mentioned in #74.
- [Canva](https://canva.com) is a graphic design platform that allows users to create social media graphics and other visual content.

## Checklist

- [X] Added icon(s) to the appropriate directory
- [X] Updated documentation with the new icon(s)
- [X] Verified the icon(s) display correctly
- [X] If the icon has light and dark, verified the auto switch works correctly

## Closing

closes #74, closes tandpfun/skill-icons#184, closes tandpfun/skill-icons#358, closes tandpfun/skill-icons#805

### CODE REVIEW

**Comment by WorldOfDex** [comment]
thanks man
, shall i expect black and white version?

**Review by WorldOfDex — COMMENTED** [review]
good

**Comment by josemateuss** [comment]
> thanks man
, shall i expect black and white version?

Hi @WorldOfDex, from what i've seen here, either we have the dark and light version or we have a single one, I don't think @LelouchFR will accept both the dark and light but the single one. What do you think @LelouchFR?

**Comment by LelouchFR** [comment]
I totally agree with @josemateuss, since it's only squares, it is better to just fill up the entire icon place instead of putting it in the middle. I would prefer to have it like right now, it makes a little bit more sense

**Comment by josemateuss** [comment]
> I totally agree with @josemateuss, since it's only squares, it is better to just fill up the entire icon place instead of putting it in the middle. I would prefer to have it like right now, it makes a little bit more sense

Agreed. So, the PR is ready for review man, thanks.

**Comment by LelouchFR** [comment]
@josemateuss I'm going to look at that tomorrow, starting to get late here (and tomorrow I have exams)

**Comment by WorldOfDex** [comment]
> > thanks man
> > , shall i expect black and white version?
> 
> Hi @WorldOfDex, from what i've seen here, either we have the dark and light version or we have a single one, I don't think @LelouchFR will accept both the dark and light but the single one. What do you think @LelouchFR?

ok thanks

File: assets/canva-auto.svg [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<svg id="canva" width="256" height="256" viewBox="0 0 256 256" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <style>
+        #canva {
+            rect {fill: #242938}
+
+        @media (prefers-color-scheme: light) {
+            rect {fill: #F4F2ED}
+            }
```
Comment (LelouchFR) [inline assets/canva-auto.svg]: there's a closing bracket without any opening one, please remove it.

**Comment by josemateuss** [comment]
@LelouchFR I added the pull request template so that we don't run out of templates until we understand how template selection works, it will help new contributors as a guideline too.

File: assets/canva-auto.svg (hunk 1 above)
Comment (LelouchFR) [reply assets/canva-auto.svg]: @josemateuss is it possible to fix this asap, so it gets merged ?

**Comment by josemateuss** [comment]
@LelouchFR have you reviewed the PR?

**Comment by LelouchFR** [comment]
> @LelouchFR have you reviewed the PR?

Yes, don't you see the comment about canva-auto.svg I made?

**Comment by josemateuss** [comment]
> Yes, don't you see the comment about canva-auto.svg I made?

No, I don't, where is it? Hahaha

**Comment by josemateuss** [comment]
I don't know what do you have commented, but I remade the auto icon, please, take a look, thanks.

**Review by LelouchFR — CHANGES_REQUESTED** [review]
please use this version for the canva-auto.svg file:

```svg
<svg width="256" height="256" viewBox="0 0 256 256" fill="none" xmlns="http://www.w3.org/2000/svg" id="canva">
<style>
#canva {
    rect {fill: #242938}

    @media (prefers-color-scheme: light) {
        rect {fill: #F4F2ED}
    }
}
</style>
<g clip-path="url(#clip0_1_15)">
<rect width="256" height="256" rx="60" fill="#242938"/>
<g clip-path="url(#clip1_1_15)">
<path d="M128 228C183.228 228 228 183.228 228 128C228 72.7715 183.228 28 128 28C72.7715 28 28 72.7715 28 128C28 183.228 72.7715 228 128 228Z"
fill="#7D2AE7"/>
<path d="M128 228C183.228 228 228 183.228 228 128C228 72.7715 183.228 28 128 28C72.7715 28 28 72.7715 28 128C28 183.228 72.7715 228 128 228Z"
fill="url(#paint0_radial_1_15)"/>
<path d="M128 228C183.228 228 228 183.228 228 128C228 72.7715 183.228 28 128 28C72.7715 28 28 72.7715 28 128C28 183.228 72.7715 228 128 228Z"
fill="url(#paint1_radial_1_15)"/>
<path d="M128 228C183.228 228 228 183.228 228 128C228 72.7715 183.228 28 128 28C72.7715 28 28 72.7715 28 128C28 183.228 72.7715 228 128 228Z"
fill="url(#paint2_radial_1_15)"/>
<path d="M171.173 148.514C170.347 148.514 169.621 149.211 168.865 150.733C160.331 168.039 145.589 180.284 128.531 180.284C108.807 180.284 96.5923 162.479 96.5923 137.881C96.5923 96.2143 119.808 72.1235 140.199 72.1235C149.727 72.1235 155.546 78.1114 155.546 87.6403C155.546 98.9497 149.121 104.938 149.121 108.926C149.121 110.717 150.235 111.801 152.443 111.801C161.316 111.801 171.729 101.605 171.729 87.2026C171.729 73.2375 159.575 62.9725 139.184 62.9725C105.485 62.9725 75.5353 94.2151 75.5353 137.443C75.5353 170.904 94.6429 193.016 124.125 193.016C155.417 193.016 173.511 161.882 173.511 151.777C173.511 149.539 172.366 148.514 171.173 148.514Z"
fill="white"/>
</g>
</g>
<defs>
<radialGradient id="paint0_radial_1_15" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse"
gradientTransform="translate(66.6325 205.264) rotate(-49.416) scale(154.683)">
<stop stop-color="#6420FF"/>
<stop offset="1" stop-color="#6420FF" stop-opacity="0"/>
</radialGradient>
<radialGradient id="paint1_radial_1_15" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse"
gradientTransform="translate(80.947 50.7364) rotate(54.703) scale(174.434)">
<stop stop-color="#00C4CC"/>
<stop offset="1" stop-color="#00C4CC" stop-opacity="0"/>
</radialGradient>
<radialGradient id="paint2_radial_1_15" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse"
gradientTransform="translate(66.6316 205.263) rotate(-45.1954) scale(152.811 70.2796)">
<stop stop-color="#6420FF"/>
<stop offset="1" stop-color="#6420FF" stop-opacity="0"/>
</radialGradient>
<clipPath id="clip0_1_15">
<rect width="256" height="256" fill="white"/>
</clipPath>
<clipPath id="clip1_1_15">
<rect width="200" height="200" fill="white" transform="translate(28 28)"/>
</clipPath>
</defs>
</svg>
```

**Comment by josemateuss** [comment]
@LelouchFR can you review? Sorry for the late, your comments weren't showing up here.

**Review by LelouchFR — CHANGES_REQUESTED** [review]
You're 4 commits behind, can you get your branch up to date ? thank you ;)

**Review by LelouchFR — APPROVED** [review]

**Pull request merged**